sky/packages/sky/lib/rendering/object.dart

```diff
@@ -427,6 +521,11 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
       });
     }
   }
+
+  /// Bootstrap the rendering pipeline by scheduling the very first layout
+  ///
+  /// Requires this render object to be attached and that this render object
+  /// is the root of the render tree.
```

Comment:
Give a pointer to the code that calls this, for people trying to track down how it all works.